templates/master/01-master-container-runtime/_base/files/container-storage.yaml

```diff
@@ -0,0 +1,120 @@
+filesystem: "root"
+mode: 0644
+path: "/etc/containers/storage.conf"
+contents:
+  inline: |
+    # storage.conf is the configuration file for all tools
```

Comment:
I don't think we need the comments in the yaml files. They sorta clutter what is actually being set.

Reply:
The comments have a lot of info on the options in the config files, that is why I have them there. The comments are lost anyway when a CR is created so not completely attached to having them. @mrunalp WDYT?

Reply:
Yeah we could get rid of them later. right now they are matching the crio.conf before the CR changes are applied. We want to change the default that is laid down is well first.